source/events/issue_handler/actions/created/commands/go/source/breaking-changes/findBreakingChanges.ts

```diff
@@ -25,15 +25,16 @@ const breakingChangesPrompt = `You are a senior software engineer with the task
 A "breaking change" is an update that:
 - Changes the behavior of an existing function or feature in a way that is not backward-compatible. This could be changes to method signatures, return values, or exception types, etc.
```

Comment:
I was reading the prompt and was wondering if "This could be changes to method signatures, return values, or exception types, etc." makes it focus too much on method signatures, return values and exception types.

You have more experience with the prompts, but to me seems that "Changes the behavior of an existing function or feature in a way that is not backward-compatible."

Maybe I would add to it instead:

"Changes the behavior of an existing function or feature in a way that is not backward-compatible and makes dependant code to stop working correctly. Breaking changes will be explicitly mentioned in the changelog usually marked by 'breaking change' or similar words that clearly state that something is a breaking change either in the change's title or description."

My further thoughts are that "breaking changes" and below "Removes or deprecates an existing function or feature." are "deprecating changes" and in changelogs the writer writes either "breaking changes" / "breaking updates" / "breaking" etc. My point is that it is explicitly told what is breaking and maybe the model should not try to infer itself if it's breaking or not - if the writer did not write that it's "breaking change", mark it as ok.

I was thinking if we can guide model to look for explicit indicators of a breaking change like "breaking change" title.

What do you think?